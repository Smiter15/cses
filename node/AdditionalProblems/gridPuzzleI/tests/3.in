50
8 10 12 8 14 16 8 12 9 11 16 17 14 9 11 8 13 12 11 12 11 15 14 15 16 8 11 12 10 14 14 9 13 10 11 10 14 15 12 14 9 15 13 11 8 11 8 9 9 10
14 10 15 11 10 11 14 12 14 11 9 11 14 12 9 16 12 11 9 12 13 12 13 12 16 13 9 8 9 12 13 14 15 13 19 10 5 10 12 10 9 7 12 11 13 12 11 9 13 10
