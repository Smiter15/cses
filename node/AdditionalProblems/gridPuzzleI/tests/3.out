..X.....X...X..X........X......XX.X...............
X.X...X........X........X.......XXX.........X...X.
......X.X...X..X....X.X.XX....XXX.X...............
X.X............X........X.........X.........XX..X.
....................XXXXXX...XXXXXX...X...X.......
X.X...XXX...XX.XX..X..............X........XXXX.X.
.............................XXXXXX...X...X.......
.............X.XXX.XXXXXXX........X...............
...X.XXXXX.XX.....................X...............
X.X...................................XX..XXXXX.XX
.................X.XXXXXXX...XXXXXXX.X............
XXXXXXXXXX.XXX.XX...............................XX
................................XXXX.XXXX.XXXXXX..
......................XXXXX.XXXX..................
...........XXXXXXXXXXX............................
...XXXXXXXX.......................................
XXX....................................XX.XXXXXXXX
..........................XXXXXXXXXX.XX...........
...............XXXXXXXXXXX........................
...XXXXXXXXXXXX...................................
XXX.......................................XXXXXXXX
..........................XXXXXXXXXX.XXXXX........
............XXXXXXXXXXXXXX........................
XXXXXXXXXXXX...................................XXX
..............................XXXXXX.XXXXXXXXXX...
......................XXXXXXXX....................
...........XXXXXXXXXXX............................
XXXXXXXXXXX......................................X
.......................................XXXXXXXXXX.
.........................XXXXXXXXXXXXXX...........
...........XXXXXXXXXXXXXX.........................
..XXXXXXXXX.......................................
XX.....................................XXXXXXXXXXX
.............................XXXXXXXXXX...........
..................XXXXXXXXXXX.....................
........XXXXXXXXXX................................
XXXXXXXX....................................XXXXXX
.............................XXXXXXXXXXXXXXX......
.................XXXXXXXXXXXX.....................
...XXXXXXXXXXXXXX.................................
XXX.........................................XXXXXX
.............................XXXXXXXXXXXXXXX......
................XXXXXXXXXXXXX.....................
.....XXXXXXXXXXX..................................
XXXXX..........................................XXX
....................................XXXXXXXXXXX...
............................XXXXXXXX..............
...................XXXXXXXXX......................
..........XXXXXXXXX...............................
XXXXXXXXXX........................................
